leakcanary-android-core/src/main/java/leakcanary/internal/HeapDumpTrigger.kt

```diff
@@ -102,6 +105,8 @@ internal class HeapDumpTrigger(
       retainedReferenceCount = objectWatcher.retainedObjectCount
     }
 
+    onRetainInstanceListener.onChange(CountChanged(retainedReferenceCount))
```

Comment:
did the count actually change?

Reply:
Not necessarily, it might be the same count and it is fine. It's used to let listener know that we just performed a retained instances check and the parameter is the current `retainedReferenceCount`.
The `CountChanged` is incorrect name, I can rename it to `MaybeCountChanged` or `CountUpdated`. Word `updated` doesn't suggest it actually changed, right?

Reply:
How about only sending CountChanged when the count actually changed?